### DISCUSSION TITLE
Change the Packaging structure

### DISCUSSION THREAD

**Discussion by yezz123** [Abdenasser/dr_scaffold#59] [Ideas]
Hey @Abdenasser what do you think about using Flit as Python packaging 📦  for `Dr_scaffold` and let the workflow static its could take less time and good for dropping the traditional way of packaging

**Comment by Abdenasser** [comment]
Yeah, I think that's a good idea @yezz123 and it will make it easier for us to roll out new versions I believe.. let me know if you need the PyPI credentials !

**Reply by yezz123** [reply]
> Yeah, I think that's a good idea @yezz123 and it will make it easier for us to roll out new versions I believe.. let me know if you need the PyPI credentials!

I will try to create a PR tonight and we could switch to this way of building and Packaging! 🚀

**Comment by leogregianin** [comment]
Nothing against flit, I see it's a very good project. But what do you think about using [python-poetry](https://github.com/python-poetry/poetry)?

**Reply by yezz123** [reply]
Yeah Poetry is good too, I use it in most of my projects and work projects, but as you know let's be specific flit is a good tool that could be used while developing a package or a library that's doesn't have many steps or a way of building it's easy and work fine in building env, but if we need managing for deps we must use poetry or pyenv or if we need a way of building and distribution its good to use setuptool
